package.json

```diff
@@ -88,9 +86,10 @@
     "prettier": "^2.7.1",
     "qunit": "^2.19.1",
     "qunit-dom": "^2.0.0",
+    "release-it": "^14.2.1",
```

Comment:
I think `create-rwjblue-release-it-setup` creates a slightly outdated setup, we should be able to use v15.